setup.py

```diff
@@ -58,7 +58,8 @@
         "requests-futures>0.9.0",
         "paramiko>1.8.0,<3.0.0",
         "requests<3.0.0",
-        "retrying",
+        "tenacity",
+        "typing-extensions>=3.7.4",
```

Comment:
Is the `py.typed` stuff coming in part 2?

Reply:
Probably a much later part. My guess is 8-10 PRs for typing this.